charts/eks/templates/syncer-deployment.yaml

```diff
@@ -169,6 +174,7 @@ spec:
           {{- end }}
           {{- if or .Values.proxy.metricsServer.nodes.enabled .Values.proxy.metricsServer.pods.enabled}}
           - --proxy-metrics-server=true
+          - --single-binary-distro=false
```

Comment:
Why do we have an env var and flag for this? Also why is this necessary?

Reply:
this info is needed at 2 places basically, one is at places like endpoints syncer, apiregistration etc.
and the other is in [specialservices](https://github.com/loft-sh/vcluster/pull/1337/files#diff-7daa660cd5210e66fb754f6f39c71db5b72775919140a81c1f376ea9cd4e0d7aR51)

Since we don't have a way to inject controller context here, I thought to inject the env var

Reply:
Also, would it make more sense to pass down the name of the distro as an arg instead of this flag and then decide within the code if it's a single binary distro?